docs/generation/gradle/validateSite.gradle

```diff
@@ -74,7 +73,8 @@ for (location in ["local", "remote"]) {
 
     classpath = configurations.nuValidator
     main = "nu.validator.client.SimpleCommandLineValidator"
-    args "--skip-non-html" // --also-check-css doesn't work (still checks css as html), so limit to html files
+    args "--also-check-css"
```

Comment:
🚀

Reply:
Just one of the many benefits of updating!